package.json

```diff
@@ -6,7 +6,7 @@
   "types": "src/index.ts",
   "scripts": {
     "test": "nyc ava",
-    "lint": "eslint . --ext .js,.ts -f codeframe && tsc --noEmit",
```

Comment:
I have removed JS files as if you have compiled files around it is not passing linting